tests/storage/__init__.py

```diff
@@ -203,8 +203,9 @@ def test_create_collection(self, requires_collections, get_storage_args,
         if getattr(self, 'dav_server', '') == 'radicale':
             pytest.skip('MKCOL is broken under Radicale 1.x')
 
-        if getattr(self, 'dav_server', '') == 'icloud':
-            pytest.skip('iCloud requires a minimum-length for collection name')
+        if getattr(self, 'dav_server', '') in \
+           ('icloud', 'fastmail', 'davical'):
+            pytest.skip('Manual cleanup would be necessary.')
```

Comment:
That's not the issue, we would need to have a `delete` method across storages, although we don't have a need for this during normal usage. Also creating and deleting collections is tested with iCloud and FastMail all the time, (because that's the only way to properly set up and tear down all state) so there's really no need for this test to be run.